client/my-sites/pages/blog-posts-page.jsx

```diff
@@ -8,13 +8,40 @@ import React from 'react';
  */
 import CompactCard from 'components/card/compact';
 import Gridicon from 'components/gridicon';
+import { isEnabled } from 'config';
 
 const BlogPostsPage = React.createClass( {
+	propTypes() {
+		return {
+			homePageType: React.PropTypes.string,
+			pageForPosts: React.PropTypes.string
```

Comment:
Should this be:
```pageForPosts: React.PropTypes.oneOfType( [
    React.PropTypes.string,
    React.PropTypes.number
] )```

...since it's sometimes `0` (per L# 22)?